code/modules/surgery/helpers.dm

```diff
@@ -118,7 +118,11 @@
 
 
 /proc/get_pain_modifier(mob/living/carbon/human/M) //returns modfier to make surgery harder if patient is conscious and feels pain
-	if(M.stat) //stat=0 if CONSCIOUS, 1=UNCONSCIOUS and 2=DEAD. Operating on dead people is easy, too. Just sleeping won't work, though.
+	if(M.stat == DEAD) // Operating on dead people is easy
+		return 1
+	var/datum/status_effect/incapacitating/sleeping/S = M.has_status_effect(STATUS_EFFECT_SLEEPING)
+	if(M.stat == UNCONSCIOUS && !(S?.voluntary))
```

Comment:
```suggestion
	if(M.stat == UNCONSCIOUS && !S?.voluntary)
```
pretty sure these aren't needed. 
https://discord.com/channels/145533722026967040/727448459862343711/984943118774308894

Reply:
The `!thing?.` syntax can be tricky and I would recommend to avoid using it, unless you're absolutely sure. For example, the goal here with `!S?.voluntary` looks to be making sure `voluntary` is `FALSE`, but it will also succeed if `S` is null. 

Now it's probably not possible to be `UNCONCIOUS` without having the sleeping status, so it's appears to be fine in this particular instance.

Reply:
They aren't needed. They are only there due to
![20220612-133616-Discord](https://user-images.githubusercontent.com/7831163/173235876-69261472-4c20-477a-84d9-15f4bfb9d21a.png)

Updated the comment to explain what's going on in further detail.